docs/Configuration.md

```diff
@@ -251,7 +251,7 @@ config key | description | value type | value default | notes
 `streets` | Include street input files (OSM/PBF) | boolean | true | 
 `storage` | Configure access to data sources like GRAPH/OSM/DEM/GTFS/NETEX/ISSUE-REPORT. | object | null | 
 `subwayAccessTime` | Minutes necessary to reach stops served by trips on routes of `route_type=1` (subway) from the street | double | 2.0 | units: minutes
-`transferRequests` | Routing requests to use for pre-calculating stop-to-stop transfers. | object | `[ { mode: WALK } ]` |
+`transferRequests` | Routing requests to use for pre-calculating stop-to-stop transfers. | array | `[ { modes: ["WALK"] } ]` |
```

Comment:
Base on `NodeAdapter#asRequestModes()` `modes` is a `string`:
https://github.com/opentripplanner/OpenTripPlanner/blob/8dc91ab84a448d31931d8355bff64908c0fb3f63/src/main/java/org/opentripplanner/standalone/config/NodeAdapter.java#L188-L191

```suggestion
`transferRequests` | Routing requests to use for pre-calculating stop-to-stop transfers. | array | `[ { modes: "WALK" } ]` |
```